CHANGELOG.md

```diff
@@ -128,6 +128,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Fixed batch size auto-scaling feature to set the new value on the correct model attribute ([#3043](https://github.com/PyTorchLightning/pytorch-lightning/pull/3043)) 
 - Fixed automatic batch scaling not working with half precision ([#3045](https://github.com/PyTorchLightning/pytorch-lightning/pull/3045))
 - Fixed setting device to root gpu ([#3042](https://github.com/PyTorchLightning/pytorch-lightning/pull/3042))
+- Fixed IoU score for classes not present in target or pred ([#3098](https://github.com/PyTorchLightning/pytorch-lightning/pull/3098))
```

Comment:
Should be moved to the `[unreleased]` section

Reply:
Ah, oops. I saw that this PR was marked as `0.9.x` earlier, and hadn't noticed 0.9.0 went out already. Moved to `[unreleased]` in https://github.com/PyTorchLightning/pytorch-lightning/pull/3098/commits/1fb0dd72701ef13e695f3559fc1eaef8a0dfc4da